src/utils/featheryClient/index.ts

```diff
@@ -802,6 +802,34 @@ export default class FeatheryClient extends IntegrationClient {
     );
   }
 
+  runServerSideLogicRule(id: string) {
+    const { userId } = initInfo();
+    const data: any = {
+      id: id,
+      form_key: this.formKey,
+      fuser_key: userId
+    };
+
+    const url = `${API_URL}logic-rule/execute/`;
+    const options = {
+      headers: { 'Content-Type': 'application/json' },
+      method: 'POST',
+      body: JSON.stringify(data),
+      keepalive: false
+    };
+
+    const run = () =>
+      this._fetch(url, options).then((response) =>
+        response ? response.json() : Promise.resolve()
+      );
+    return this.offlineRequestHandler.runOrSaveRequest(
```

Comment:
let's not pass this through the offline request handler. let's just call _fetch directly